fn/context_guard.go

```diff
@@ -173,9 +173,10 @@ func (g *ContextGuard) Create(ctx context.Context,
 	return ctx, cancel
 }
 
-// ctxQuitUnsafe spins off a goroutine that will block until the passed context
-// is cancelled or until the quit channel has been signaled after which it will
-// call the passed cancel function and decrement the wait group.
+// ctxQuitUnsafe increases the wait group counter, waits until the context is
+// cancelled and decreases the wait group counter. It stores the passed cancel
+// function and returns a wrapped version, which removed the stored one and
+// calls it. Quit method calls all the stored cancel functions.
```

Comment:
nit: either `The Quit method calls...` or just `Quit calls ...`

Reply:
Thanks! I changed it to "The Quit method calls ...".
Also fixed another instance in the comment below:
```go
// We don't have to wait on g.quit here: g.quit can be closed only in
// the Quit method, which also closes the context we are waiting for.
context.AfterFunc(ctx, func() {
        g.wg.Done()
})
```